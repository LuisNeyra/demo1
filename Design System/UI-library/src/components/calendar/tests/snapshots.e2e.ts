import { snapIt } from '../../../test';

const component = 'calendar';

describe(component, () => {
  const snap = snapIt(component);

  describe('renders', () => {
    snap('a default calendar', '<blaze-calendar date="1982, January 14"></blaze-calendar>');
    snap('a coloured calendar', '<blaze-calendar type="info" date="1982, January 14"></blaze-calendar>');
  });
});
