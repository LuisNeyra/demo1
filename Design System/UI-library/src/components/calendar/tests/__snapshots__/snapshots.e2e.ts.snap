// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`calendar renders a coloured calendar 1`] = `"<blaze-calendar type=\\"info\\" date=\\"1982, January 14\\" class=\\"hydrated\\"><div class=\\"c-calendar\\"><button class=\\"c-calendar__control c-calendar__control--prev-year\\">‹</button><div class=\\"c-calendar__header c-calendar__header--year\\">1982</div><button class=\\"c-calendar__control c-calendar__control--next-year\\">›</button><button class=\\"c-calendar__control c-calendar__control--prev-month\\">‹</button><div class=\\"c-calendar__header c-calendar__header--month\\">January</div><button class=\\"c-calendar__control c-calendar__control--next-month\\">›</button><div class=\\"c-calendar__day\\">Su</div><div class=\\"c-calendar__day\\">Mo</div><div class=\\"c-calendar__day\\">Tu</div><div class=\\"c-calendar__day\\">We</div><div class=\\"c-calendar__day\\">Th</div><div class=\\"c-calendar__day\\">Fr</div><div class=\\"c-calendar__day\\">Sa</div><button class=\\"c-calendar__date c-button\\" aria-selected=\\"false\\">27</button><button class=\\"c-calendar__date c-button\\" aria-selected=\\"false\\">28</button><button class=\\"c-calendar__date c-button\\" aria-selected=\\"false\\">29</button><button class=\\"c-calendar__date c-button\\" aria-selected=\\"false\\">30</button><button class=\\"c-calendar__date c-button\\" aria-selected=\\"false\\">31</button><button class=\\"c-calendar__date c-calendar__date--in-month c-button\\" aria-selected=\\"false\\">1</button><button class=\\"c-calendar__date c-calendar__date--in-month c-button\\" aria-selected=\\"false\\">2</button><button class=\\"c-calendar__date c-calendar__date--in-month c-button\\" aria-selected=\\"false\\">3</button><button class=\\"c-calendar__date c-calendar__date--in-month c-button\\" aria-selected=\\"false\\">4</button><button class=\\"c-calendar__date c-calendar__date--in-month c-button\\" aria-selected=\\"false\\">5</button><button class=\\"c-calendar__date c-calendar__date--in-month c-button\\" aria-selected=\\"false\\">6</button><button class=\\"c-calendar__date c-calendar__date--in-month c-button\\" aria-selected=\\"false\\">7</button><button class=\\"c-calendar__date c-calendar__date--in-month c-button\\" aria-selected=\\"false\\">8</button><button class=\\"c-calendar__date c-calendar__date--in-month c-button\\" aria-selected=\\"false\\">9</button><button class=\\"c-calendar__date c-calendar__date--in-month c-button\\" aria-selected=\\"false\\">10</button><button class=\\"c-calendar__date c-calendar__date--in-month c-button\\" aria-selected=\\"false\\">11</button><button class=\\"c-calendar__date c-calendar__date--in-month c-button\\" aria-selected=\\"false\\">12</button><button class=\\"c-calendar__date c-calendar__date--in-month c-button\\" aria-selected=\\"false\\">13</button><button class=\\"c-calendar__date c-calendar__date--in-month c-button c-calendar__date--selected c-button--info\\" aria-selected=\\"true\\">14</button><button class=\\"c-calendar__date c-calendar__date--in-month c-button\\" aria-selected=\\"false\\">15</button><button class=\\"c-calendar__date c-calendar__date--in-month c-button\\" aria-selected=\\"false\\">16</button><button class=\\"c-calendar__date c-calendar__date--in-month c-button\\" aria-selected=\\"false\\">17</button><button class=\\"c-calendar__date c-calendar__date--in-month c-button\\" aria-selected=\\"false\\">18</button><button class=\\"c-calendar__date c-calendar__date--in-month c-button\\" aria-selected=\\"false\\">19</button><button class=\\"c-calendar__date c-calendar__date--in-month c-button\\" aria-selected=\\"false\\">20</button><button class=\\"c-calendar__date c-calendar__date--in-month c-button\\" aria-selected=\\"false\\">21</button><button class=\\"c-calendar__date c-calendar__date--in-month c-button\\" aria-selected=\\"false\\">22</button><button class=\\"c-calendar__date c-calendar__date--in-month c-button\\" aria-selected=\\"false\\">23</button><button class=\\"c-calendar__date c-calendar__date--in-month c-button\\" aria-selected=\\"false\\">24</button><button class=\\"c-calendar__date c-calendar__date--in-month c-button\\" aria-selected=\\"false\\">25</button><button class=\\"c-calendar__date c-calendar__date--in-month c-button\\" aria-selected=\\"false\\">26</button><button class=\\"c-calendar__date c-calendar__date--in-month c-button\\" aria-selected=\\"false\\">27</button><button class=\\"c-calendar__date c-calendar__date--in-month c-button\\" aria-selected=\\"false\\">28</button><button class=\\"c-calendar__date c-calendar__date--in-month c-button\\" aria-selected=\\"false\\">29</button><button class=\\"c-calendar__date c-calendar__date--in-month c-button\\" aria-selected=\\"false\\">30</button><button class=\\"c-calendar__date c-calendar__date--in-month c-button\\" aria-selected=\\"false\\">31</button><button class=\\"c-calendar__date c-button\\" aria-selected=\\"false\\">1</button><button class=\\"c-calendar__date c-button\\" aria-selected=\\"false\\">2</button><button class=\\"c-calendar__date c-button\\" aria-selected=\\"false\\">3</button><button class=\\"c-calendar__date c-button\\" aria-selected=\\"false\\">4</button><button class=\\"c-calendar__date c-button\\" aria-selected=\\"false\\">5</button><button class=\\"c-calendar__date c-button\\" aria-selected=\\"false\\">6</button><div class=\\"c-calendar__footer\\"><button class=\\"c-calendar__today c-button c-button--block c-button--info\\">Today</button></div></div></blaze-calendar>"`;

exports[`calendar renders a default calendar 1`] = `"<blaze-calendar date=\\"1982, January 14\\" class=\\"hydrated\\"><div class=\\"c-calendar\\"><button class=\\"c-calendar__control c-calendar__control--prev-year\\">‹</button><div class=\\"c-calendar__header c-calendar__header--year\\">1982</div><button class=\\"c-calendar__control c-calendar__control--next-year\\">›</button><button class=\\"c-calendar__control c-calendar__control--prev-month\\">‹</button><div class=\\"c-calendar__header c-calendar__header--month\\">January</div><button class=\\"c-calendar__control c-calendar__control--next-month\\">›</button><div class=\\"c-calendar__day\\">Su</div><div class=\\"c-calendar__day\\">Mo</div><div class=\\"c-calendar__day\\">Tu</div><div class=\\"c-calendar__day\\">We</div><div class=\\"c-calendar__day\\">Th</div><div class=\\"c-calendar__day\\">Fr</div><div class=\\"c-calendar__day\\">Sa</div><button class=\\"c-calendar__date c-button\\" aria-selected=\\"false\\">27</button><button class=\\"c-calendar__date c-button\\" aria-selected=\\"false\\">28</button><button class=\\"c-calendar__date c-button\\" aria-selected=\\"false\\">29</button><button class=\\"c-calendar__date c-button\\" aria-selected=\\"false\\">30</button><button class=\\"c-calendar__date c-button\\" aria-selected=\\"false\\">31</button><button class=\\"c-calendar__date c-calendar__date--in-month c-button\\" aria-selected=\\"false\\">1</button><button class=\\"c-calendar__date c-calendar__date--in-month c-button\\" aria-selected=\\"false\\">2</button><button class=\\"c-calendar__date c-calendar__date--in-month c-button\\" aria-selected=\\"false\\">3</button><button class=\\"c-calendar__date c-calendar__date--in-month c-button\\" aria-selected=\\"false\\">4</button><button class=\\"c-calendar__date c-calendar__date--in-month c-button\\" aria-selected=\\"false\\">5</button><button class=\\"c-calendar__date c-calendar__date--in-month c-button\\" aria-selected=\\"false\\">6</button><button class=\\"c-calendar__date c-calendar__date--in-month c-button\\" aria-selected=\\"false\\">7</button><button class=\\"c-calendar__date c-calendar__date--in-month c-button\\" aria-selected=\\"false\\">8</button><button class=\\"c-calendar__date c-calendar__date--in-month c-button\\" aria-selected=\\"false\\">9</button><button class=\\"c-calendar__date c-calendar__date--in-month c-button\\" aria-selected=\\"false\\">10</button><button class=\\"c-calendar__date c-calendar__date--in-month c-button\\" aria-selected=\\"false\\">11</button><button class=\\"c-calendar__date c-calendar__date--in-month c-button\\" aria-selected=\\"false\\">12</button><button class=\\"c-calendar__date c-calendar__date--in-month c-button\\" aria-selected=\\"false\\">13</button><button class=\\"c-calendar__date c-calendar__date--in-month c-button c-calendar__date--selected\\" aria-selected=\\"true\\">14</button><button class=\\"c-calendar__date c-calendar__date--in-month c-button\\" aria-selected=\\"false\\">15</button><button class=\\"c-calendar__date c-calendar__date--in-month c-button\\" aria-selected=\\"false\\">16</button><button class=\\"c-calendar__date c-calendar__date--in-month c-button\\" aria-selected=\\"false\\">17</button><button class=\\"c-calendar__date c-calendar__date--in-month c-button\\" aria-selected=\\"false\\">18</button><button class=\\"c-calendar__date c-calendar__date--in-month c-button\\" aria-selected=\\"false\\">19</button><button class=\\"c-calendar__date c-calendar__date--in-month c-button\\" aria-selected=\\"false\\">20</button><button class=\\"c-calendar__date c-calendar__date--in-month c-button\\" aria-selected=\\"false\\">21</button><button class=\\"c-calendar__date c-calendar__date--in-month c-button\\" aria-selected=\\"false\\">22</button><button class=\\"c-calendar__date c-calendar__date--in-month c-button\\" aria-selected=\\"false\\">23</button><button class=\\"c-calendar__date c-calendar__date--in-month c-button\\" aria-selected=\\"false\\">24</button><button class=\\"c-calendar__date c-calendar__date--in-month c-button\\" aria-selected=\\"false\\">25</button><button class=\\"c-calendar__date c-calendar__date--in-month c-button\\" aria-selected=\\"false\\">26</button><button class=\\"c-calendar__date c-calendar__date--in-month c-button\\" aria-selected=\\"false\\">27</button><button class=\\"c-calendar__date c-calendar__date--in-month c-button\\" aria-selected=\\"false\\">28</button><button class=\\"c-calendar__date c-calendar__date--in-month c-button\\" aria-selected=\\"false\\">29</button><button class=\\"c-calendar__date c-calendar__date--in-month c-button\\" aria-selected=\\"false\\">30</button><button class=\\"c-calendar__date c-calendar__date--in-month c-button\\" aria-selected=\\"false\\">31</button><button class=\\"c-calendar__date c-button\\" aria-selected=\\"false\\">1</button><button class=\\"c-calendar__date c-button\\" aria-selected=\\"false\\">2</button><button class=\\"c-calendar__date c-button\\" aria-selected=\\"false\\">3</button><button class=\\"c-calendar__date c-button\\" aria-selected=\\"false\\">4</button><button class=\\"c-calendar__date c-button\\" aria-selected=\\"false\\">5</button><button class=\\"c-calendar__date c-button\\" aria-selected=\\"false\\">6</button><div class=\\"c-calendar__footer\\"><button class=\\"c-calendar__today c-button c-button--block\\">Today</button></div></div></blaze-calendar>"`;
